### PR TITLE
chore: Publish preview release + deploy staging every 3 hours

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: .github/workflows/trigger-staging-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          curl -X POST \
+          -H "Authorization: Bearer ${{ secrets.STAGING_DEPLOY_ACCESS_TOKEN }}" \
+          -H "Accept: application/vnd.github.v3+json" \
+          https://api.github.com/repos/@medusajs/staging/actions/workflows/deploy.yml/dispatches \
```
Comment (srindom) [inline .github/workflows/trigger-staging-deployment.yml]: shouldn't this be?
```suggestion
          https://api.github.com/repos/medusajs/staging/actions/workflows/deploy.yml/dispatches \
```

Comment (olivermrbl) [reply .github/workflows/trigger-staging-deployment.yml]: yes nice catch!

**Comment by adrien2p** [comment]
Question, couldn't we deploy everytime we merge and cancel the previous deployment if we have many merge one after each other? It might be a dumb question ahah

**Comment by olivermrbl** [comment]
> Question, couldn't we deploy everytime we merge and cancel the previous deployment if we have many merge one after each other? It might be a dumb question ahah

Long-term, yes, I think we should do this. However, right now, we are merging quite a lot, so the schedule is just a measure to not overload the system 😅

**Comment by adrien2p** [comment]
makes total sense 👍

**Review by adrien2p — APPROVED** [review]

**Pull request merged**